lib/web_ui/test/canvaskit/text_fragmenter_test.dart

```diff
@@ -19,23 +17,23 @@ void testMain() {
 
   group('$fragmentUsingIntlSegmenter', () {
     test('fragments text into words', () {
-      final Uint32List breaks = fragmentUsingIntlSegmenter(
+      final SkUint32List breaks = fragmentUsingIntlSegmenter(
         'Hello world 你好世界',
         IntlSegmenterGranularity.word,
       );
       expect(
-        breaks,
+        breaks.toTypedArray(),
         orderedEquals(<int>[0, 5, 6, 11, 12, 14, 16]),
       );
     });
 
     test('fragments multi-line text into words', () {
-      final Uint32List breaks = fragmentUsingIntlSegmenter(
+      final SkUint32List breaks = fragmentUsingIntlSegmenter(
```

Comment:
In these tests, we may want to actually free these objects afterward. CanvasKit normally will free these for you after you pass them into the APIs, but since they aren't actually being passed into a CanvasKit API, we may be on the hook to free them ourselves. Probably doesn't matter that much for tests, but we are probably technically leaking memory in the CanvasKit module.

Reply:
Good idea!